cmds/exp/disk_unlock/disk_unlock.go

```diff
@@ -1,42 +1,26 @@
-// Copyright 2020 the u-root Authors. All rights reserved
+// Copyright 2022 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// The disk_unlock command is used to unlock a disk drive as follows:
```

Comment:
why was this comment changed so much.

Reply:
The code to compute the password that this comment describes was moved to hsskey.go, so I also moved the comment to that file.